idaes/generic_models/unit_models/plate_heat_exchanger.py

```diff
@@ -179,86 +182,89 @@ def build(self):
-        # (E-NTU method) and inluded as performance equations. Hence the control
-        #  volume energy balances are not added.
+        # (E-NTU method) and inluded as performance equations. Hence the
+        # control volume energy balances are not added.
 
-        # ======================================================================
+        # =====================================================================
         # Build cold-side  Control Volume(Rich solvent)
-        self.cold_side = ControlVolume0DBlock(default={
+        self.cold_fluid = ControlVolume0DBlock(default={
             "dynamic": self.config.dynamic,
             "has_holdup": self.config.has_holdup,
             "property_package": self.config.cold_side.property_package,
-            "property_package_args": self.config.cold_side.property_package_args})
+            "property_package_args":
+                self.config.cold_side.property_package_args})
 
-        self.cold_side.add_state_blocks(has_phase_equilibrium=False)
+        self.cold_fluid.add_state_blocks(has_phase_equilibrium=False)
 
-        self.cold_side.add_material_balances(
+        self.cold_fluid.add_material_balances(
             balance_type=MaterialBalanceType.componentTotal,
             has_mass_transfer=False,
             has_phase_equilibrium=False,
             has_rate_reactions=False)
 
-        self.cold_side.add_momentum_balances(
+        self.cold_fluid.add_momentum_balances(
             balance_type=MomentumBalanceType.pressureTotal,
             has_pressure_change=True)
 
-        # ======================================================================
+        # =====================================================================
         # Add Ports to control volumes
         # hot-side
         self.add_inlet_port(name="hot_inlet",
-                            block=self.hot_side, doc='inlet Port')
+                            block=self.hot_fluid, doc='inlet Port')
         self.add_outlet_port(name="hot_outlet",
-                             block=self.hot_side, doc='outlet Port')
+                             block=self.hot_fluid, doc='outlet Port')
 
         # cold-side
         self.add_inlet_port(name="cold_inlet",
-                            block=self.cold_side, doc='inlet Port')
+                            block=self.cold_fluid, doc='inlet Port')
         self.add_outlet_port(name="cold_outlet",
-                             block=self.cold_side, doc='outlet Port')
-        # ======================================================================
+                             block=self.cold_fluid, doc='outlet Port')
+        # =====================================================================
         # Add performace equation method
         self._make_params()
         self._make_performance_method()
 
     def _make_params(self):
         self.P = Param(initialize=self.config.passes,
```

Comment:
The names for these need to be more descriptive. `P` is an especially poor choice as it invites confusion with pressure.

Also, does this actually need to be a `Param`? If it is only used for building the sets below, then I don't think it is necessary and we could just use the config argument directly.